etc/visual-studio/UnitTests.vcxproj

```diff
@@ -50,6 +50,7 @@
         MBEDTLS_CONFIG_FILE="mbedtls-config.h";
         OPENTHREAD_CONFIG_FILE="openthread-windows-config.h";
         OPENTHREAD_MULTIPLE_INSTANCE;
+        OPENTHREAD_FTD=1;
```

Comment:
I actually tried putting it in the header first, but it did not work.

There are a few combinations of include file order where 'openthread-config.h' not the first file being included, and ... because of the domino-chain of includes it did not work.

You can see what I mean if you move it back to the header.
That's a different problem that should be solved on a different PR/ticket.